incubator/factorio/README.md

```diff
@@ -0,0 +1,71 @@
+# Factorio
+
+[Factorio](https://www.factorio.com/) is a game in which you build and maintain factories.
+
+## Introduction
+
+This chart creates a single [Factorio Headless](https://www.factorio.com/download-headless) Pod, plus Services for the Factorio server and RCON.
+
+## Prerequisites
+
+- Kubernetes 1.4+ with Beta APIs enabled
+- PV provisioner support in the underlying infrastructure
+
+## Provider-specific Prerequisites
+
+Amazon's Elastic Loadbalancer lacks support for UDP. You'll need to set `factorioServer.ServiceType` to `NodePort` and expose the port that gets selected (see `kubectl describe svc <servicename>`) via a security group. You may need to do something similar for certain bare metal deployments.
```

Comment:
Can we add a quick line about this in NOTES.txt, and point to this for more info?